extras/gomock.bzl

```diff
@@ -55,14 +55,15 @@ def _gomock_source_impl(ctx):
         aux_files = []
         for target, pkg in ctx.attr.aux_files.items():
             f = target.files.to_list()[0]
-            aux = ctx.actions.declare_file(paths.join(gopath, "src", pkg, f.basename))
+            aux = ctx.actions.declare_file(paths.join("gopath", "src", pkg, f.basename))
             ctx.actions.run_shell(
                 outputs = [aux],
                 inputs = [f],
                 command = "mkdir -p {0} && cp -L {1} {0}".format(aux.dirname, f.path),
             )
             aux_files.append("{0}={1}".format(pkg, aux.path))
             needed_files.append(f)
```

Comment:
Do you know whether this line is needed? Maybe adding `f` instead of `aux` was just a typo back then?

Reply:
Hm, yeah, you could be right. Let me test if swapping `s/f/aux/` will be enough here. Thanks for the suggestion!

Reply:
Looks like the `needed_files.append(f)` is redundant here, indeed.